src/auto/key_file.rs

```diff
@@ -283,7 +293,12 @@ impl KeyFile {
         }
     }
 
-    //pub fn to_data(&self) -> Result<(String, /*Unimplemented*/Fundamental: Size), Error> {
-    //    unsafe { TODO: call ffi::g_key_file_to_data() }
-    //}
+    pub fn to_data(&self) -> Result<(String, usize), Error> {
```

Comment:
This one is also suboptimal. The usize is the length of the string, it should just return Result<String, Error>. Also going to be fixed with my array-length changes.

But this one here is at least not unsafe, only suboptimal :)